autoendpoint/src/error.rs

```diff
@@ -58,6 +58,9 @@ pub enum ApiErrorKind {
     #[error(transparent)]
     PayloadError(actix_web::Error),
 
+    #[error("Payload Error: {0}")]
+    PayloadErrorS(String),
```

Comment:
Do we need this? I don't see it being used

Reply:
Ah, good catch! I missed that.